File: android/calc/app/src/main/java/com/example/minwook/calc/MainActivity.java
package com.example.minwook.calc;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void add_click(View v){
        EditText a = findViewById(R.id.number1);
        EditText b = findViewById(R.id.number2);

        TextView result = findViewById(R.id.result);

        int n1 = Integer.parseInt(a.getText().toString());
        int n2 = Integer.parseInt(b.getText().toString());
        result.setText(Integer.toString(n1+n2));
    }

    public void sub_click(View v){
        EditText a = findViewById(R.id.number1);
        EditText b = findViewById(R.id.number2);

        TextView result = findViewById(R.id.result);

        int n1 = Integer.parseInt(a.getText().toString());
        int n2 = Integer.parseInt(b.getText().toString());
        result.setText(Integer.toString(n1-n2));
    }
    public void mul_click(View v){
        EditText a = findViewById(R.id.number1);
        EditText b = findViewById(R.id.number2);

        TextView result = findViewById(R.id.result);

        int n1 = Integer.parseInt(a.getText().toString());
        int n2 = Integer.parseInt(b.getText().toString());
        result.setText(Integer.toString(n1*n2));
    }
    public void div_click(View v){
        EditText a = findViewById(R.id.number1);
        EditText b = findViewById(R.id.number2);

        TextView result = findViewById(R.id.result);

        int n1 = Integer.parseInt(a.getText().toString());
        int n2 = Integer.parseInt(b.getText().toString());
        result.setText(Integer.toString(n1/n2));
    }
}

File: android/kotlin_test/app/src/main/java/com/example/minwook/kotlin_test/MainActivity.kt
package com.example.minwook.kotlin_test

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var arr = arrayListOf<Int>()

        result.text = "0"
        saveB.text ="saved number"

        number_text.setText(result.text)

        add_button.setOnClickListener {
            var r:Int = number_text.text.toString().toInt()
            r+=1
            result.text = r.toString()
            number_text.setText(result.text)
        }
        remove_button.setOnClickListener{
            var r:Int = number_text.text.toString().toInt()
            r-=1
            result.text = r.toString()
            number_text.setText(result.text)
        }

        saveB.setOnClickListener {
            arr.add(result.text.toString().toInt())
            printarr.text = arr.toString()
        }
        r.setOnClickListener {
            arr.clear()
            printarr.text ="clear!"
        }
    }
}
